packages/ava/src/data/statistics/index.ts

```diff
@@ -0,0 +1,269 @@
+/**
+ * Statistical methods used internally by ava.
+ */
+
+import { assert } from '../utils';
+
+import * as cache from './caches';
+
+/**
+ * Calculate the minimum of the array.
+ * @param value - The array to process
+ */
+export function min(value: number[]): number {
+  const cachedValue = cache.get<number>(value, 'min');
+  if (cachedValue !== undefined) {
+    return cachedValue;
+  }
+  return cache.set(value, 'min', Math.min(...value));
+}
+
+function calculateMinIndex(value: number[]) {
+  let min = value[0];
+  let idx = 0;
+  for (let i = 0; i < value.length; i += 1) {
+    if (value[i] < min) {
+      idx = i;
+      min = value[i];
+    }
+  }
+  return idx;
+}
+
+/**
+ * Calculate the minimum index of the array.
+ * @param value - The array to process
+ */
```

Comment:
```suggestion
/**
 * Calculate the minimum index of the array.
 * ---
 * 得到数组中最小值的 index
 *
 * @param value - The array to process
 */
```

could add some Chinese comments for some jsdocs